core/sdk-core/src/main/java/software/amazon/awssdk/core/SdkSystemSetting.java

```diff
@@ -170,8 +169,7 @@
     AWS_MAX_ATTEMPTS("aws.maxAttempts", null),
 
     /**
-     * Which {@link DefaultsMode} to use, case insensitive
-     * @see DefaultsMode
+     * Which {@code DefaultsMode} to use, case insensitive
      */
     AWS_DEFAULTS_MODE("aws.defaultsMode", null),
```

Comment:
Does this still belong in `sdk-core`?

Reply:
We don't actually have an `AwsSystemSetting`. AWS related system properties such as `AWS_ACCESS_KEY_ID` are in this class already, so I kept it here to be consistent.